src/main/interface/msp.c

```diff
@@ -523,10 +523,11 @@ static bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProce
         break;
 
     case MSP_ANALOG:
-        sbufWriteU8(dst, (uint8_t)constrain((getBatteryVoltage() / 10), 0, 255));
+        sbufWriteU8(dst, (uint8_t)constrain((getBatteryVoltage() + 5) / 10, 0, 255));
         sbufWriteU16(dst, (uint16_t)constrain(getMAhDrawn(), 0, 0xFFFF)); // milliamp hours drawn from battery
         sbufWriteU16(dst, getRssi());
         sbufWriteU16(dst, (int16_t)constrain(getAmperage(), -0x8000, 0x7FFF)); // send current in 0.01 A steps, range is -320A to 320A
+        sbufWriteU16(dst, (uint16_t)constrain(getBatteryVoltage(), 0, 0xFFFF));
```

Comment:
No need for the `constrain()` here - `getBatteryVoltage()` is already an `uint16_t`.